build.gradle

```diff
@@ -183,7 +184,7 @@ ext{
     configureSecurityPlugin = { OpenSearchCluster cluster ->
 
         // Retrieve Security Plugin Zip from zipArchive
-        configurations.zipArchive.asFileTree.each {
+        configurations.secureIntegTestPluginArchive.asFileTree.each {
             if(it.name.contains("opensearch-security")) {
```

Comment:
Just a nit, we wont need this name check since only security is in the `secureIntegTestPluginArchive`